openapi-spec/swagger.yaml

```diff
@@ -0,0 +1,534 @@
+      tags:
+        - API versions
+      description: Gets version details by specified API version.
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: object
+            properties:
+              version:
+                $ref: '#/definitions/VersionSpec'
+          examples:
+            application/json:
+              name: v1beta
+              status: SUPPORTED
+              updated: '2017-04-10T14:36:58.014Z'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+
+  '/v1beta/{tenantId}/orchestration/services':
+    parameters:
+      - $ref: '#/parameters/tenantId'
+    get:
+      tags:
+        - Service Definitions
+      description: Lists information for all Service Defintions.
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/ServiceDefs'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+    post:
+      tags:
+        - Service Definitions
+      description: Registers a Service Definition.
+      parameters:
+        - name: Definition name
+          in: body
+          schema:
+            $ref: '#/definitions/ServiceDefsReq'
+      responses:
+        '202':
+          description: Accepted
+          schema:
+            $ref: '#/definitions/ServiceDefs'
+        '400':
+          $ref: '#/responses/HTTPStatus400'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{tenantId}/orchestration/instances':
+    parameters:
+      - $ref: '#/parameters/tenantId'
+    get:
+      tags:
+        - Service Instance
+      description: Gets all service instances detail.
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceSpecs'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+    post:
+      tags:
+        - Service Instance
+      description: Create an instance of service definition.
+      parameters:
+        - name: Instance details
+          in: body
+          schema:
+            $ref: '#/definitions/InstanceReq'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceCreationResp'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+    delete:
+      tags:
```

Comment:
delete url should be: /v1beta/{tenantId}/orchestration/instances/{instanceId} and there is no request body in delete method

Reply:
Right. Had misplaced the position in swagger. API behaves accordingly